puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-release-test.yml

```diff
@@ -0,0 +1,31 @@
+---
+- job-template:
+    name: foreman-{foreman_version}-{distro}-release-test
+    project-type: 'pipeline'
+    sandbox: true
+    parameters:
+      - string:
+          name: expected_version
+          default: 'UNKNOWN'
```

Comment:
you could make the default saner by passing it as `{foreman_version}` or even better define the expected version in the project definition.

see https://github.com/SatelliteQE/robottelo-ci/blob/8d758d2a12b17da0fc23e344fcc1d4747dfe388b/jobs/release/sat-release.yaml#L48-L60 for an example

Reply:
But we don't know the exact version. Remember that this is in the form of 1.23.0-RC1. That's the whole point of this parameter.